### PR TITLE
Website/add metadata

### PR DESCRIPTION
**Pull request by xuechunL**
add some meta-data for auto deploy website.

### CODE REVIEW

File: op-guide/binary-deployment.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+---
+title: TiDB Binary 部署方案详解
+category: 安装 & 部署
```
Comment (andelf) [inline op-guide/binary-deployment.md]: better not use non-ascii in category.

Comment (xuechunL) [inline op-guide/binary-deployment.md]: ok

File: trouble-shooting.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+---
+title: TiDB 集群故障诊断
+category: Advanced Usage
```
Comment (andelf) [inline trouble-shooting.md]: !

File: op-guide/docker-compose.md [hunk 3]
```diff
@@ -1,3 +1,8 @@
+---
+title: 使用 Docke Compose 构建集群
```
Comment (andelf) [inline op-guide/docker-compose.md]: typo

**Comment by GregoryIan** [comment]
LGTM

**Pull request merged**